### DISCUSSION TITLE
How do you change DataTable.Pagination dropdown menu background color

### DISCUSSION THREAD

**Discussion by kyusungpark** [callstack/react-native-paper#4083] [General]
Hi, I am tyring to change the background color for DataTable.Pagination's dropdown menu. I went through the dock and Pagination and looked for all the properties that can take a ColorValue and didn't find one that works. 
Also tried to update the background color using style prop, but this changes the color of the whole pagination.

Any ideas?
<img width="318" alt="Screenshot 2023-09-17 at 6 28 09 PM" src="https://github.com/callstack/react-native-paper/assets/58890841/96f5c715-0c0b-4711-b4a5-75780fea81cd">

**Comment by GninninwokyOuattara** [comment]
The dropdown background color is handled by the level2 property from elevation.
If you provide your own theme, you can modify it there. Otherwise you can do it like this

```javascript
<DataTable.Pagination
      theme={{
          colors: {
              elevation: {
                  level2: "green",
              },
          },
      }}
        
/>
```
![image](https://github.com/callstack/react-native-paper/assets/29814522/6d571a35-b98d-4977-9e7d-83c4e029cdee)

**Reply by botjaeger** [reply]
I know that this is an old post, but how did you know? do they provide documentation for this?
I also need to customize DataTable.Pagination by changing its font size from labels to menu dropdown items

**Reply by GninninwokyOuattara** [reply]
I was faced with the same issue and couldn't find the documentation either. Settled for navigating the code to figure it out.
